### ISSUE TITLE
relational sort

### ISSUE THREAD

**Issue by piotzkhider** [z-song/laravel-admin#328]
```php
class LogController extends Controller
{
    /**
     * Index interface.
     *
     * @return Content
     */
    public function index()
    {
        return Admin::content(function (Content $content) {
            $content->header(trans('admin::lang.operation_log'));
            $content->description(trans('admin::lang.list'));

            $grid = Admin::grid(OperationLog::class, function (Grid $grid) {
                $grid->model()->orderBy('id', 'DESC');

                $grid->id('ID')->sortable();
                $grid->user()->name()->sortable(); // <-
```

why not work?

**Comment by z-song** [comment]
It seems to be bug, It works fine for `HasOne` relation but `BelongsTo` relation not

I'll fix this later.

**Comment by z-song** [comment]
@piotzkhider issue fixed, switch to version 1.3.x-dev and try again.

**Comment by xuanchau99** [comment]
> It seems to be bug, It works fine for `HasOne` relation but `BelongsTo` relation not
> 
> I'll fix this later.

2021: have you fixed it?
